docs/install_guides/pypi-openvino-dev.md

```diff
@@ -78,20 +84,32 @@ python -m pip install --upgrade pip
 
 ### Step 4. Install the Package
 
-Run the command below: <br>
+To install and configure components of the development package for working with specific frameworks, use the `pip install openvino-dev[extras]` command, where `extras` is a list of extras from the table below: 
+
+| DL Framework                                                                     | Extra                          |
+| :------------------------------------------------------------------------------- | :-------------------------------|
+|   [Caffe*](https://caffe.berkeleyvision.org/)                                    | caffe                           |
+|   [Caffe* 2](https://caffe2.ai/)                                                 | caffe2                          |
+|   [Kaldi*](https://kaldi-asr.org/)                                               | kaldi                           |
+|   [MXNet*](https://mxnet.apache.org/)                                            | mxnet                           |
+|   [ONNX*](https://github.com/microsoft/onnxruntime/)                             | onnx                            |
+|   [PyTorch*](https://pytorch.org/)                                               | pytorch                         |
+|   [TensorFlow* 1.x](https://www.tensorflow.org/versions#tensorflow_1)            | tensorflow                      |
+|   [TensorFlow* 2.x](https://www.tensorflow.org/versions#tensorflow_2)            | tensorflow2                     |
 
+For example, to install and configure components for working with TensorFlow 2.x, MXNet and Caffe, use the folowing command:  
```

Comment:
configure components --> configure the components
use the folowing command --> use the following command